src/include/sof/samples/audio/kwd_nn_detect_test.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Cristina Feies <cristina.ilie@nxp.com>
+ * Author: Viorel Suman <viorel.suman@nxp.com>
+ */
+
+#ifndef __USER_KWD_NN_DETECT_TEST_H__
+#define __USER_KWD_NN_DETECT_TEST_H__
+
+#include <sof/samples/audio/kwd_nn/kwd_nn_config.h>
+
+#define KWD_NN_SILENCE         0
+#define KWD_NN_UNKNOWN         1
+#define KWD_NN_YES_KEYWORD     2
+#define KWD_NN_NO_KEYWORD      3
+
+#define N               2
```

Comment:
This needs a more descriptive and prefixed name.

Reply:
I gave up to N - it could be used to compute the size of sliding window we move on audio data flow to check if there is a keyword in the next amount of data. 

However we fixed this window for the moment at the half of the keyword length size, it is good enough for the algorithm.